content/en/docs/installation/armory-operator/_index.md

```diff
@@ -0,0 +1,87 @@
+
+## Advantages to using a Kubernetes Operator for Spinnaker deployment
+
+* Use a Kubernetes manifest to deploy and manage Spinnaker or Armory Enterprise.
+* Use `kubectl` to deploy, manage, and access Spinnaker or Armory Enterprise like you would with any other app deployed on Kubernetes.
+* Store and reference Spinnaker secrets in one of the [supported secrets engines]({{< ref "secrets" >}}).
+* Store your configuration in `git` for an auditable and reversible GitOps workflow.
+
+## How the Spinnaker Operator and the Armory Operator work
+
+The Operator is a custom controller of Kubernetes kind `Deployment`. The Operator works with a [Kubernetes custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) named `SpinnakerService` to deploy and manage Spinnaker in your cluster.
+
+### `SpinnakerService` Custom Resource Definition
+
+The `SpinnakerService` CRD declares the resource's names, the scope of the resource, and the structural schema of the `spinnakerservice.yml` [Kubernetes manifest](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/) that contains your Spinnaker configuration.
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: spinnakerservices.spinnaker.io
+spec:
+  group: spinnaker.io
+  names:
+    kind: SpinnakerService
+    listKind: SpinnakerServiceList
+    plural: spinnakerservices
+    shortNames:
+    - spinsvc
+    singular: spinnakerservice
+  scope: Namespaced
+```
+
+You can view the entire `SpinnakerService` CRD in the `spinnaker-operator` [repo](https://github.com/armory/spinnaker-operator/blob/master/deploy/crds/spinnaker.io_spinnakerservices_crd.yaml).
+
+### Spinnnaker manifest
+
+You have the following options for creating the Spinnaker manifest:
+
+* Create the manifest YAML file yourself using an example `spinnakerservice.yml` as a starting point.
+* Create a collection of patch files that [Kustomize](https://kustomize.io/) uses to overwrite sections of the `spinnakerservice.yml` manifest. Kustomize is part of `kubectl`.
+
+Regardless of the option you use to configure Spinnaker, the `spinnakerservice.yml` manifest specifies that the Kubernetes kind is `SpinnakerService`.
+
+```yaml
+apiVersion: spinnaker.io/{{< param "operator-oss-crd-version" >}}
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+```
+
+You can view the rest of the basic file in the `spinnaker-operator` [repo](https://github.com/armory/spinnaker-operator/blob/master/deploy/spinnaker/basic/spinnakerservice.yml).
+
+At the highest level, when you use `kubectl` to apply the manifest:
+
+1. `kubectl` parses `spinnakerservice.yml` and sees that the kind is `SpinnakerService`.
+2. `kubectl` delegates manifest validation and deployment to the Spinnaker Operator.
+3. The Operator validates the manifest and uses Halyard to deploy Spinnaker.
```

Comment:
```suggestion
1. `kubectl` parses `spinnakerservice.yml` and sees that the kind is `SpinnakerService`.
1. `kubectl` delegates manifest validation and deployment to the Spinnaker Operator.
1. The Operator validates the manifest and uses Halyard to deploy Spinnaker.
```

Using `1.` for each numbered step still renders sequentially, but allows for the addition, removal, and re-ordering of steps without having to adjust each step. I suggest using this as standard practive.

Reply:
I agree. My editor, however, automatically numbers sequentially, and I didn't catch it to correct this time.